### DISCUSSION TITLE
Name override for pharaoh's sceptre

### DISCUSSION THREAD

**Discussion by AndrewCreevey** [runelite/runelite#16446] [Ideas]
An override that changes the names of the pharaoh's sceptre teleports to make it easier to remember what goes where

**Changes:**
- Jalsavrah -> Pyramid plunder
- Jaleustrophos -> Agility pyramid
- Jaldraocht -> Ancient pyramid
- Jaltevas -> Necropolis / tombs of amascut

**Comment by Nightfirecat** [comment]
[The Easy Teleports hub plugin](https://runelite.net/plugin-hub/show/easy-pharaoh-sceptre) does this along with other confusingly-named teleports.
